### ISSUE TITLE
NER can be quite off sometimes

### ISSUE THREAD

**Issue by computron** [materialsintelligence/matscholar#52]
e.g. the abstract for https://www.researchgate.net/publication/273349137_Realization_of_narrowband_thermal_emission_with_optical_nanostructures

seems to have a lot of inaccuracies

**Comment by computron** [comment]
another example:
https://link.springer.com/article/10.1007/s11708-009-0009-x

**Comment by LeighWeston86** [comment]
Looks like most of these errors are related to the application tag, which is one of our worst performing tags. There aren't a lot of papers like these in our NER training set.

**Comment by jppgks** [comment]
@LeighWeston86 is the NER training set available for download somewhere?

**Comment by computron** [comment]
Yes the training set is here: https://figshare.com/articles/Materials_Science_Named_Entity_Recognition_train_development_test_sets/8184428

**Comment by jppgks** [comment]
Thanks @computron!

**Comment by littleflow3r** [comment]
@LeighWeston86 is it possible to share the code used for training? i tried to reproduce the result but i only got like 76% accuracy, while in the paper it is mentioned abt 87%.
Thanks!
